mne/epochs.py

```diff
@@ -343,13 +343,16 @@ def preload_data(self):
         assert self._data.shape[-1] == len(self.times)
         return self
 
-    def decimate(self, decim, copy=False):
+    def decimate(self, decim, offset=0, copy=False):
         """Decimate the epochs
 
         Parameters
         ----------
         decim : int
             The amount to decimate data.
+        offset : int
+            Apply an offset to where the decimation starts.
+            The offset is in samples, at the original sampling rate.
```

Comment:
I think you need to say "at the current sampling rate". I think of the "original" as being for the Raw instance, and I could have done e.g. `epochs.decim(2, copy=False)` before another call with `epochs.decim(2, copy=False, offset=1)` -- and for that second call, the sample rate of the `epochs` object would be at half the original (raw) sample rate.